src/tiffIO.cpp

```diff
@@ -160,12 +160,21 @@ tiffIO::tiffIO(char *fname, DATA_TYPE newtype) {
 	if (datatype == SHORT_TYPE) {
```

Comment:
These may be fixes that are really a separate issue.  If we are OK that this is the case, and tests all work, then OK to merge.